cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -47,3 +47,20 @@ ifeq ($(PYTHON_LIBDIR),None)
 endif
 
 PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_inc() )')
+
+# For tcl scripts purpose source files paths in form of C:/User/..
```

Comment:
VERILOG_SOURCES and the other variables in this files are not only used by TCL script; they are sometimes directly passed to simulators, and used as dependency in Makefiles. Is it just the comment that's wrong, and msys paths should (or can) always be Windows-style paths with forward slashes?  Can you adjust the comment accordingly?